server/src/routes/compliance.ts

```diff
@@ -1,14 +1,27 @@
+import multer from 'multer';
+import path from 'path';
 import express from 'express';
 import PaginationMiddleware from '../middlewares/paginationMiddleware';
 import { limiter } from '../config/index';
 
 const buildComplianceRoutes = (controller: any) => {
   const router = express.Router();
 
-  router.get('/compliance/list', limiter, controller.getAllComplianceReports);
+  const storage = multer.diskStorage({
+    destination: (_req, _file, cb) => cb(null, 'uploads/'),
+    filename: (_req, file, cb) => cb(null, `${file.fieldname}-${Date.now()}${path.extname(file.originalname)}`)
+  });
```

Comment:
could you change 'cb' for something more descriptive?

Reply:
"done" will be more descriptive than "cb". Changed.